BitwardenShared/UI/Tools/Send/SendItem/AddEditSendItem/AddEditSendItemView.swift

```diff
@@ -11,6 +11,12 @@ struct AddEditSendItemView: View { // swiftlint:disable:this type_body_length
     /// The `Store` for this view.
     @ObservedObject var store: Store<AddEditSendItemState, AddEditSendItemAction, AddEditSendItemEffect>
 
+    /// A state variable to track whether the TextField is focused
+    @FocusState private var isMaxAccessCountFocused: Bool
+
+    /// A state variable to store the text in the TextField.
+    @SwiftUI.State private var maximumAccessCountText: String = ""
```

Comment:
🤔 Can't `store. maximumAccessCountText` be used instead of duplicating the same here?

Reply:
I thought so too but given the use case where a user can enter 0, we need the ability to show that "0" in the text field.  It needs to be blank by default tho so I was having trouble getting that exact logic to work without having two variables since the TextField itself can handle entering the text vs just showing whats currently in the state.  Do you have a better idea on how to accomplish that?

Reply:
I guess you can use `store.binding` like in [here](https://github.com/bitwarden/ios/blob/main/BitwardenShared/UI/Auth/CreateAccount/CreateAccountView.swift#L78) and move all the logic to the processor, where you can use the binding property as a `string` and then have an `int` calculated property based on the string one.

Reply:
Yeah, I did try that initially but I couldn't figure out how to allow the user to enter in "0" and  also allow it be blank by default.  Since the count is 0 by default.  It seemed everything I tried that some part of the use case requirements would break.  I can keep messing with other ideas to get it to work but it seems like its one of those bizarre situations where not all of the logic can be abstracted from the View. 

```
the maximum access count is blank on default, then displays the count when edited. If the users sets it back to 0 after editing, it displays 0.
```

Reply:
If you make `maximumAccessCountText` in the state a variable rather than a computed property, you could default it to the empty string, but update it whenever `maximumAccessCount` changes. Then it would initially be empty, but once you change it to 0 or a number, it would be updated to reflect the count. Would that match the behavior you're looking for?

Reply:
yeah i can try that!  good call.